PhoneNumberKit/PhoneNumber.swift

```diff
@@ -66,7 +73,7 @@ public extension PhoneNumber {
     - Parameter rawNumber: String to be parsed to phone number struct.
     */
     @available(*, unavailable, message: "use PhoneNumberKit instead to produce PhoneNumbers")
-    public init(rawNumber: String) throws {
+    init(rawNumber: String) throws {
```

Comment:
Was Swift 5.0 complaining about these access modifiers?  Judging by the `@available` attributes, I'm assuming they're supposed to be internal anyway.  Just curious.

Reply:
Yup!